drc_cmis/admin.py

```diff
@@ -37,11 +84,33 @@ class CMISConfigAdmin(SingletonModelAdmin):
         ),
     )
 
-    def cmis_enabled(self, obj):
-        return settings.CMIS_ENABLED
+    class Media:
+        js = ("drc_cmis/js/cmis_config.js",)
+
+    def get_urls(self):
+        urls = super().get_urls()
+        extra_urls = [
+            path(
+                "api/connection",
+                self.admin_site.admin_view(self.cmis_connection_view),
+                name="cmisconfig_api_connection",
+            )
+        ]
+        return extra_urls + urls
+
+    def cmis_connection_view(self, request):
+        return CMISConnectionJSONView.as_view(model_admin=self)(request)
+
+    def cmis_enabled(self, obj=None):
+        return getattr(settings, "CMIS_ENABLED", True)
```

Comment:
As we discussed, since the CMIS_ENABLED is not a `drc_cmis` setting, this should be moved to open-zaak.

Reply:
Moved from this library to Open Zaak, here: https://github.com/open-zaak/open-zaak/pull/676/commits/ae9c3a77860a846084760ded1102f5aaa41bdb0d